api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/claim/ClaimService.java

```diff
@@ -53,38 +49,29 @@
     public ClaimResult createOrUpdateClaim(ClaimRequest claimRequest) {
```

Comment:
We should rename this method, and the similarly named method in the claim controller

Reply:
Added TODO to rename in it's own PR.